modules/administration_manual/pages/installation/manual_installation.adoc

```diff
@@ -374,6 +374,14 @@ To use the graphical Installation Wizard refer to xref:installation/installation
 
 IMPORTANT: Please know that ownCloud’s data directory *must be exclusive to ownCloud* and not be modified manually by any other process or user.
 
+[[headers]]
+== Headers
+
+NOTE: ownCloud has a mechanism to set headers programatically. These headers are set with the `always` directive to avoid
+errors when there are additionally headers set in `http.conf`. More information on Apache
```

Comment:
Please change "when there are additionally headers set in `http.conf`" to "when there are additional headers set in `http.conf`"

Reply:
fixed